README.md

```diff
@@ -123,6 +123,39 @@ For more Vertex AI samples, please visit the [Vertex AI samples Github repositor
 </table>
 <!-- markdownlint-enable MD033 -->
 
+## Related Repositories
+
+- [Google Cloud Applied AI Engineering](https://github.com/GoogleCloudPlatform/applied-ai-engineering-samples)
+- [Generative AI for Marketing using Google Cloud](https://github.com/GoogleCloudPlatform/genai-for-marketing)
+- [Generative AI for Developer Productivity](https://github.com/GoogleCloudPlatform/genai-for-developers)
+- Vertex AI Core
+  - [Vertex AI Samples](https://github.com/GoogleCloudPlatform/vertex-ai-samples)
+  - [MLOps with Vertex AI](https://github.com/GoogleCloudPlatform/mlops-with-vertex-ai)
+  - [Developing NLP solutions with T5X and Vertex AI](https://github.com/GoogleCloudPlatform/t5x-on-vertex-ai)
+  - [AlphaFold batch inference with Vertex AI Pipelines](https://github.com/GoogleCloudPlatform/vertex-ai-alphafold-inference-pipeline)
+  - [Serving Spark ML models using Vertex AI](https://github.com/GoogleCloudPlatform/vertex-ai-spark-ml-serving)
+  - [Sensitive Data Protection (Cloud DLP) for Vertex AI Generative Models (PaLM2)](https://github.com/GoogleCloudPlatform/Sensitive-Data-Protection-for-Vertex-AI-PaLM2)
+- Conversational AI
+  - [Contact Center AI Samples](https://github.com/GoogleCloudPlatform/contact-center-ai-samples)
+  - [Reimagining Customer Experience 360](https://github.com/GoogleCloudPlatform/dialogflow-ccai-omnichannel)
```

Comment:
IMO this repo, while an awesome example of the full CCAI stack, is not relevant to Generative AI.